db/migrate/20171208142645_convert_collection_belongs_to_many.rb

```diff
@@ -0,0 +1,8 @@
+class ConvertCollectionBelongsToMany < ActiveRecord::Migration
+  def change
+    create_table :collections_projects, id: false do |t|
```

Comment:
not having an id makes it hard to switch to an AR model for internal use.

Reply:
Would you prefer a `has_many :collections, through: `? I was wondering, and couldn't think of any property we might stick on the join model.

Reply:
I'm not sure really i find myself often trying to figure out the state of the classification -> subjects and have to write out the join statement  instead of a where on `ClassificationSubjects.where(subject_id: 1)`. If we have no need to write that `CollectionProject.where(collection_id: 1)` query then this point is moot.